protos/ray.proto

```diff
@@ -167,6 +169,13 @@ message SchedulerInfoReply {
   CompGraph computation_graph = 6; // The computation graph constructed so far
 }
 
+message isReadyRequest {
+  uint64 objectid = 1;
```

Comment:
This should probably be `repeated uint64 objectid = 1;` so that we can process multiple objectids in the same request. Also, include a comment.